ncvoter/settings.py

```diff
@@ -74,9 +74,9 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'ncvoter',
-        'USER': 'ncvoter',
-        'PASSWORD': 'ncvoter',
+        'NAME': 'ncvotes',
+        'USER': '',
+        'PASSWORD': '',
         'HOST': '127.0.0.1',
         'PORT': '5432',
```

Comment:
Can we take out HOST and PORT? If HOST is set to 127.0.0.1, then postgres uses 'TCP' rather than Unix domain sockets, which, on our Caktus laptops, requires a password.